core/src/main/scala/com/nvidia/spark/rapids/tool/ToolTextFileWriter.scala

```diff
@@ -34,55 +35,44 @@ class ToolTextFileWriter(
     finalLocationText: String,
     hadoopConf: Option[Configuration] = None) extends Logging {
 
-  // use same as Spark event log writer
-  val LOG_FILE_PERMISSIONS = new FsPermission(Integer.parseInt("660", 8).toShort)
-  val LOG_FOLDER_PERMISSIONS = new FsPermission(Integer.parseInt("770", 8).toShort)
-  private val textOutputPath = new Path(s"$finalOutputDir/$logFileName")
-  private val hadoopConfToUse = hadoopConf.getOrElse(RapidsToolsConfUtil.newHadoopConf)
-
-  private val defaultFs = FileSystem.getDefaultUri(hadoopConfToUse).getScheme
-  private val isDefaultLocal = defaultFs == null || defaultFs == "file"
-  private val uri = textOutputPath.toUri
+  private val textOutputPath = s"$finalOutputDir/$logFileName"
```

Comment:
so I think the convention if call it "Path" if its a Path type, if not maybe call this textOutputLoc

Reply:
Changed it to `textOutputLoc`